src/main/resources/data/spectrum/recipes/mod_integration/the_grande_spectacular_mod_compat_double_feature/malum_create/anvil_crushing/brilliance_crushing.json

```diff
@@ -0,0 +1,22 @@
+{
+  "type": "spectrum:anvil_crushing",
+  "ingredient": {
+    "item": "malum:brilliant_stone"
+  },
+  "crushedItemsPerPointOfDamage": 0.75,
+  "experience": 0.75,
+  "result": {
+    "item": "malum:crushed_brilliance"
+  },
+  "particleEffectIdentifier": "enchanted_hit",
+  "soundEventIdentifier": "block.metal.break",
+  "fabric:load_conditions": [
+    {
+      "condition": "fabric:all_mods_loaded",
+      "values": [
+        "create",
```

Comment:
wait, why does this require create to be present? Is crushed_brilliance a malum<->create compat item?

Reply:
Yep, this and crushed soulstone are create compat items for crushing malum ores in a create crushing wheel.